<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scp360::ModifySharedFolderDialog</class>
 <widget class="QDialog" name="scp360::ModifySharedFolderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify shared folder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="_volumeNameLabel">
     <property name="text">
      <string>Volume name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="_volumeNameLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="_hostPathLabel">
     <property name="text">
      <string>Host path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="_hostPathLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="_browsePushButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="_buttonBoxSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>scp360::ModifySharedFolderDialog</receiver>
   <slot>_accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>scp360::ModifySharedFolderDialog</receiver>
   <slot>_reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_browsePushButton</sender>
   <signal>clicked()</signal>
   <receiver>scp360::ModifySharedFolderDialog</receiver>
   <slot>_browsePushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_browsePushButtonClicked()</slot>
  <slot>_accept()</slot>
  <slot>_reject()</slot>
 </slots>
</ui>
